docs/introduction.rst

```diff
@@ -0,0 +1,521 @@
+can be defined using four parameters:
+
+1. Maximum anomaly. If this is set then the flag is set to 1, fail if the absolute difference between the value and
+   the climatological average at that point is greater than the maximum anomaly, otherwise it is set to 0, pass.
+2. If standard_deviation is set then the value is converted to a standardised anomaly. the flag is set to 1, fail if
+   the absolute standardised anomaly is greater than the maximum anomaly, otherwise it is set to 0, pass.
+3. If standard_deviation_limits is set then the input standard deviation is constrained to lie between the upper and
+   lower limits thus specified before the calculation of the standardised anomalies.
+4. If lowbar is set then the absolute anomaly must be greater than the lowbar to fail regardless of the standard
+   deviation.
+
+These allow for a great deal of flexibility in the check depending what information is available.
+
+:func:`.do_supersaturation_check`
+=================================
+
+Tests whether a report represents supersaturated conditions.
+
+Check whether the dewpoint temperature is greater than the air temperature. If the dew point is greater than the
+air temperature then the conditions are supersaturated and the flag is set to 1, fail. If the dewpoint is less than
+or equal to the air temperature then the flag is set to 0, pass. If either of the inputs is numerically invalid then
+the flag is set to 2, untestable.
+
+:func:`.do_sst_freeze_check`
+============================
+
+Tests whether the sea-surface temperature is above freezing.
+
+Check whether the sea-surface temperature is above a specified freezing point (generally sea water freezes at -1.8C).
+There are optional inputs, which allow you to specify an observational uncertainty and a multiplier. If these are not
+supplied then the uncertainty is set to zero. If the sea-surface temperature is more than the multiplier times the
+uncertainty below the freezing point then the flag is set to 1, fail, otherwise it is set to 0, pass. If any of the
+inputs is numerically invalid (Nan, None or something of that kind) then the flag is set to 2, untestable.
+
+:func:`.do_wind_consistency_check`
+==================================
+
+Tests that wind speed and direction are consistent.
+
+Compares the wind speed and wind direction to check for consistency. If the windspeed is zero, the direction should
+be set to zero also. If the wind speed is greater than zero then the wind directions should not equal zero. If either
+of these constraints is violated then the flag is set to 1, fail, otherwise it is set to 0. If either of the inputs
+is numerically valid then the flag is set to 2, untestable.
+
+Running Multiple Individual Report Checks
+-----------------------------------------
+
+Multiple indvidual report checks can be run simultaneously using the :func:`.do_multiple_row_check` function. Aside from the
```

Comment:
```suggestion
Multiple individual report checks can be run simultaneously using the :func:`.do_multiple_row_check` function. Aside from the
```

Reply:
A little hint: to commit the changes you can simply click the `Commit suggestion` button.